data.py

```diff
@@ -58,7 +74,6 @@
                 "name": sy.Str(),
                 "email": sy.Email(),
                 }),
-            sy.Optional("c4dt_work"): sy.Str(),
             "tags": sy.Seq(sy.Str()),
```

Comment:
Why not use an enumerate type for the tags, now that we want to enforce a fixed list? This would provide an automatic check for consistency.

Reply:
I suppose the tag list is changing much more frequently than the category or any other list. So I'm hesitant to put an often changing list here. What do you think?

Reply:
OK, added all the tags which showed some wrong ones...